packages/storybook8/stories/Components/IncomingCallNotification/IncomingCallNotification/snippets/IncomingCallNotificationStyling.snippet.tsx

```diff
@@ -0,0 +1,56 @@
+import { IncomingCallNotification, useTheme } from '@azure/communication-react';
+import React from 'react';
```

Comment:
Let's put import react at the first line since some IDE/bundlers ask developers to do so

Reply:
We have a linter rule in SB somewhere that forces this import order

Reply:
The rule is about importing our library? I am wondering why it is doing it...